guides/common/modules/proc_puppet-prerequisites.adoc

```diff
@@ -0,0 +1,29 @@
+[id="puppet_guide_prerequisites_{context}"]
+= Prerequisites for Puppet
+
+There are four requirements to run the Puppet agent on managed hosts:
+
+. Sync the Puppet agent from https://puppet.com/docs/puppet/6.21/install_puppet.html#enable_the_puppet_platform_repository[puppet.com].
+For example, https://yum.puppet.com/puppet6/el/7/x86_64/[Puppet 6 for CentOS 7 on x86_64] or https://apt.puppet.com/[Puppet 6 for Ubuntu 20.04] with release `focal`, component `puppet6`, and architecture `amd64`.
++
+Manage these repositories by using products, sync plans, content views, composite content views, and lifecycle environments.
+See {ContentManagementDocURL} for more information.
+. Provide the Puppet agent to the managed host using their activation key.
+. Adjust the provisioning templates to have the Puppet agent installed automatically when provisioning a host:
```

Comment:
I think our default provisioning templates should be OK. Why is it needed to modify this?